agent/config/config.go

```diff
@@ -280,6 +280,8 @@ func environmentConfig() (Config, error) {
 
 	dockerStopTimeout := getDockerStopTimeout()
 
+	cgroupMemorySubsystemPath := os.Getenv("ECS_CGROUP_MEMORY_SUBSYSTEM_PATH")
```

Comment:
This needs to be updated in the readme file.

Reply:
fixed.